umm-spec-lib/src/cmr/umm_spec/validation/umm_spec_validation_utils.clj

```diff
@@ -13,6 +14,18 @@
         {field-path [(format "%%s must be unique. This contains duplicates named [%s]."
                              (str/join ", " duplicate-names))]}))))
 
+(defn date-in-past-validator
+  "Validate that the date is in the past"
+  [field-path value]
+  (when (and value (not (date/is-past? value)))
+    {field-path ["Date should be in the past."]}))
+
+(defn date-in-future-validator
```

Comment:
This isn't used yet, but there is a ticket next sprint that would use it. I figured it was easy to add while I was doing date in past.